Kitodo/src/main/java/org/kitodo/services/data/WorkpieceService.java

```diff
@@ -238,6 +247,11 @@ public void addAllObjectsToIndex() throws CustomResponseException, InterruptedEx
         indexer.performMultipleRequests(findAll(), workpieceType);
     }
 
+    @Override
+    public WorkpieceDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
+        return new WorkpieceDTO();
```

Comment:
Please add a TODO annotation that mentioned that this method is not full implemented.